benches/comparative.rs

```diff
@@ -0,0 +1,86 @@
+#[macro_use]
+extern crate criterion;
+
+#[macro_use]
+extern crate yoga;
+
+extern crate stretch;
+
+use criterion::Criterion;
+use criterion::Fun;
+use yoga::prelude::*;
+use yoga::Node;
+use yoga::StyleUnit::{Auto, UndefinedValue};
+
+fn yoga(n: u64) {
+    let mut node = Node::new();
+
+    let mut child = Node::new();
+    let mut other_child = Node::new();
+
+    node.insert_child(&mut child, 0);
+    node.insert_child(&mut other_child, 1);
+
+    style!(node,
```

Comment:
Change the root node styling to the following or just leave it without styling. The reason is that Stretch currently does not take into account outer-size of postion effecting properties on the root such as `margin` or `left`. This is something that can easily be added in the future but for now remove it to keep them consistent.

```
style!(node,
         PaddingHorizontal(4 pt),
     );
```